### PR TITLE
replaced 0.3 with master in master branch

### PR DESCRIPTION
**Pull request by The-EDev**
This is to more accurately distinguish where a crow build is coming from (it used to be the latest release, which is not the best idea)

It's worth mentioning that the changes here need to be made for each release

### CODE REVIEW

File: include/crow/app.h [hunk 1]
```diff
@@ -120,7 +120,7 @@ namespace crow
             return *this;
         }
 
-        ///Set the server name (default Crow/0.3)
+        ///Set the server name (default Crow/master)
```
Comment (mrozigor) [inline include/crow/app.h:123]: I think that you can remove info about default value. Or change to something like `Crow/<VERSION>`

File: include/crow/app.h [hunk 2]
```diff
@@ -360,7 +360,7 @@ namespace crow
     private:
         uint16_t port_ = 80;
         uint16_t concurrency_ = 1;
-        std::string server_name_ = "Crow/0.3";
+        std::string server_name_ = "Crow/master";
```
Comment (mrozigor) [inline include/crow/app.h:363]: Maybe create new header with some constexpr `VERSION_STRING` and use it here and in `http_server.h`. And then change this version from `master` to appropriate in cmake script on `deploy`/`package` target? I could prepare something like this maybe tomorrow if you want :)

File: include/crow/app.h (hunk 1 above)
Comment (The-EDev) [reply include/crow/app.h:123]: My bad, I just changed every `0.3` to master without looking much.

File: include/crow/app.h (hunk 2 above)
Comment (The-EDev) [reply include/crow/app.h:363]: That sounds great

**Comment by The-EDev** [comment]
replaced with #162

**Pull request closed without merging**